granule_ingester/setup.py

```diff
@@ -24,7 +24,7 @@
     __version__ = f.readline()
 
 try:
-    check_call(['conda', 'install', '-y', '-c', 'conda-forge', '--file', 'conda-requirements.txt'])
+    check_call(['mamba', 'install', '-y', '-c', 'conda-forge', '--file', 'conda-requirements.txt'])
```

Comment:
Any reason we're not using conda if mamba fails like we do [here](https://github.com/apache/incubator-sdap-nexus/blob/master/analysis/setup.py#L24-L31)?

Reply:
I've never had mamba fail, and from what I've read, if mamba doesn't work, conda won't either 
I did that fallback cause I didn't know that at the time

Reply:
On second thought... maybe I should still do that in the event it's being run locally w/o mamba installed